community.md

```diff
@@ -37,6 +53,24 @@ We’re happy to help! Come engage with us on any channel that works for you:
    </div>
 </div>
 
+## Report issues
+
+Report an issue/bug in the below GitHub repos and tell us if you hit a bump on the road. We want to fix all bugs!
+
+   - compiler, runtime, standard library, or tooling: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
+   - language specification: <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.
+   - website: <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo. 
+   - security flaw: send an email to security@ballerina.io. For details, see the <a href="https://ballerina.io/security/">security policy</a>.
+
+## Help us grow 
+
+Getting a language established is like getting a new religion going. If you like the experience so far, help us to spread the word.
+
+- Start by starring the <a href="https://github.com/ballerina-platform/ballerina-lang">main GitHub repo</a>. Thank you!

```

Comment:
It seems the comment [1] is not addressed even though it is marked as resolved. 

[1] https://github.com/ballerina-platform/ballerina-dev-website/pull/177#discussion_r363678508

Reply:
Done.